build.rs

```diff
@@ -1,27 +1,46 @@
 extern crate bindgen;
+use std::{env, path::PathBuf};
 
-use std::env;
-use std::path::{PathBuf, Path};
+fn ndk_home() -> PathBuf {
+    match env::var("NDK_HOME") {
+        Ok(home) => home.into(),
+        Err(_) => {
+            let android_home: PathBuf = env::var("ANDROID_HOME")
+                .expect("Failed to get ANDROID_HOME")
+                .into();
+            android_home.join("ndk/22.1.7171670")
+        }
+    }
+}
 
-fn main() {
-    let android_home = env::var("ANDROID_HOME").expect("ANDROID_HOME not set!");
-    let ndk_include_dir = Path::new(android_home.as_str()).join("ndk/21.3.6528147/sysroot/usr/include/");
-    let gl31h = ndk_include_dir.join("GLES3/gl31.h");
-    let gl31h = gl31h.to_str().unwrap();
-    let ndk_include_dir = ndk_include_dir.to_str().unwrap();
+fn android_setup() {
```

Comment:
I don't like that you commented out all code for generating the bindings, but I understand that it unreasonable to expect a user targeting iOS to have the NDK installed. I think that a better approach would be to include a small bash script to run bindgen and regenerate the headers.

Reply:
Fixed